src/commands/utils/kubectl.ts

```diff
@@ -107,53 +134,68 @@ export async function getResources<T>(
             namespaceFlags = "";
             break;
         default:
-            namespaceFlags = `-n ${namespace}`
+            namespaceFlags = `-n ${namespace}`;
             break;
     }
 
-    const labelFlags = Object.keys(labels).map(l => `-l ${l}=${labels[l]}`)
+    const labelFlags = Object.keys(labels).map((l) => `-l ${l}=${labels[l]}`);
 
-    const command = [
-        `get ${resourceName}`,
-        namespaceFlags,
-        labelFlags,
-        "-o json"
-    ].filter(arg => arg).join(" ");
+    const command = [`get ${resourceName}`, namespaceFlags, labelFlags, "-o json"].filter((arg) => arg).join(" ");
 
     const listResult = await getKubectlJsonResult<K8sList<T>>(kubectl, kubeConfigFile, command);
-    return map(listResult, r => r.items);
+    return map(listResult, (r) => r.items);
 }
 
 interface K8sList<T> {
-    items: T[]
+    items: T[];
 }
 
-export async function streamKubectlOutput(kubectl: APIAvailable<KubectlV1>, kubeConfigFile: string, kubectlArgs: string[]): Promise<Errorable<OutputStream>> {
-    const kubectlInternal = <KubectlInternal>(<any>kubectl.api).kubectl;
+export async function streamKubectlOutput(
+    kubectl: APIAvailable<KubectlV1>,
+    kubeConfigFile: string,
+    kubectlArgs: string[],
+): Promise<Errorable<OutputStream>> {
+    const kubectlInternal = asInternal(kubectl.api);
```

Comment:
Replaced `any` cast with `asInternal` function that returns an `errorable` if the underlying implementation changes.